src/components/percentage-selector/percentage-selector.tsx

```diff
@@ -0,0 +1,97 @@
+import { TouchableOpacity } from '@gorhom/bottom-sheet';
+import { BigNumber } from 'bignumber.js';
+import React, { FC, useState, useEffect } from 'react';
+import { Keyboard, KeyboardAvoidingView, Text, View } from 'react-native';
+
+import { isAndroid } from '../../config/system';
+import { formatSize } from '../../styles/format-size';
+import { usePercentageSelectorStyles } from './percentage-selector.styles';
+
+interface Props {
+  balance: string;
+  symbol: string;
+  handleChange: (newValue: BigNumber) => void;
+}
+
+export const PercentageSelector: FC<Props> = ({ symbol, balance, handleChange }) => {
+  const percentageStyles = usePercentageSelectorStyles();
+
+  const [isKeyboardVisible, setKeyboardVisible] = useState(false);
+
+  useEffect(() => {
+    const keyboardDidShowListener = Keyboard.addListener('keyboardDidShow', () => {
+      setKeyboardVisible(true);
+    });
+    const keyboardDidHideListener = Keyboard.addListener('keyboardDidHide', () => {
+      setKeyboardVisible(false);
+    });
+
+    return () => {
+      keyboardDidHideListener.remove();
+      keyboardDidShowListener.remove();
+    };
+  }, []);
+
+  return (
+    <KeyboardAvoidingView
+      keyboardVerticalOffset={isKeyboardVisible ? (isAndroid ? formatSize(80) : formatSize(37.5)) : 0}
```

Comment:
const androidKeyboardVerticalOffset = isAndroid ? formatSize(80) : formatSize(37.5); + useMemo. And reuse it here